src/utils/ICS24Host.sol

```diff
@@ -33,7 +32,7 @@ library ICS24Host {
         pure
         returns (bytes memory)
     {
-        return abi.encodePacked("commitments/channels/", channelId, "/sequences/", Strings.toString(sequence));
+        return abi.encodePacked(channelId, uint8(1), uint64ToBigEndian(sequence));
```

Comment:
Does big endian make any difference in encode packed? If `encodePacked` is using little endian by default, could we ask ibc-go to use little endian instead?

Reply:
We can check if this would be fine with spec @AdityaSripal @sangier, but this is the current ibc-go implementation, so I suggest we do that change in a separate issue in that case.

Reply:
Generally its a bit better for it to be bigEndian since it allows us to reverseIterate in order. So ordered trees will have better iterators with this key format